.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - master
+      - 82-interactive-cli
+
+jobs:
+  publish-docker-image:
+    name: Publish Docker image
```

Comment:
Additional questions, but PR is good to be merged:
* Does this publish the image generated from `Dockerfile` into github? or is it another unrelated thing?
* If it's the image from `Dockerfile`, how would we use it later in the different platforms?

Reply:
I think giving the user the option to choose might be enough. We can just give them options like this:
```
* Dockerfile9: php8.1, composer 2, npm 16...
* Dockerfile: php7.4, composer 1, npm 12...
* Choose your own image: this might not work out of the box. You'll be prompted to type it.
```

Then with that info, we can search/replace the config file of the platform they've used.

Reply:
Yep! I like that.